recipe/tango-include-dirs.patch

```diff
@@ -0,0 +1,13 @@
+diff --git a/CMakeLists.txt b/CMakeLists.txt
+index 5f96806..f36f348 100644
+--- a/CMakeLists.txt
++++ b/CMakeLists.txt
+@@ -26,7 +26,7 @@ set(HEADERS
+ link_directories(${TANGO_PKG_LIBRARY_DIRS})
+ 
+ add_executable(${PROJECT_NAME} ${SOURCES} ${HEADERS})
+-target_include_directories(${PROJECT_NAME} PUBLIC ${CMAKE_CURRENT_SOURCE_DIR} ${TANGO_PKG_INCLUDE_DIRS})
++target_include_directories(${PROJECT_NAME} PUBLIC ${CMAKE_CURRENT_SOURCE_DIR} ${TANGO_PKG_INCLUDE_DIRS} ${TANGO_PKG_INCLUDE_DIRS}/tango)
```

Comment:
@t-b due to [MR 424](https://gitlab.com/tango-controls/cppTango/-/merge_requests/924) that I included in the new cpptango conda build (see https://github.com/conda-forge/cpptango-feedstock/pull/9), I had to add `${TANGO_PKG_INCLUDE_DIRS}/tango`. 

I guess it's because we don't use `pkgconfig` on windows. I have to pass both `%LIBRARY_INC%` and `%LIBRARY_INC%\tango`. That sounds correct?
If it does, I can make a MR in the TangoTest repo.

Reply:
Sounds correct!